model.py

```diff
@@ -7504,6 +7524,14 @@ class Credential(Base):
         UniqueConstraint('data_source_id', 'patron_id', 'type'),
     )
 
+
+    # A meaningless identifier used to identify this patron (and no other)
+    # to a remote service.
+    IDENTIFIER_TO_REMOTE_SERVICE = "Identifier Sent To Remote Service"
+
+    # An identifier used by a remote service to identify this patron.
+    IDENTIFIER_FROM_REMOTE_SERVICE = "Identifier Received From Remote Service"
+
```

Comment:
This is not used in this branch but it is used by the RBdigital branch I'm working on.